src/popover/popover.jsx

```diff
@@ -304,29 +305,68 @@ const Popover = React.createClass({
     }
   },
 
+  getOverlapMode(anchor, target, median) {
+    if ([anchor, target].indexOf(median) >= 0) return 'auto';
+    if (anchor === target) return 'inclusive';
+    return 'exclusive';
+  },
+
+  getPositions(anchor, target) {
+    let a = Extend(anchor, {});
```

Comment:
no, it creates a new object, and copies from `anchor` onto the new object, and then copies from `{}` onto the same one